content/docs/authorization/configuration/proxy-server/_index.md

```diff
@@ -14,9 +14,23 @@ The storage administrator must first configure Authorization with the following
 - Roles
 - Role bindings
 
+The address of the Authorization proxy-server must be specified when executing `karavictl`.
+
+For the `RPM deployment`, the address is the DNS-hostname of the machine where the RPM is installed.
+
+For the `Helm/Operator deployment`, the address is exposed via LoadBalancer/NodePort by the Ingress Controller consuming the proxy-server Ingress. By default, this is the NGINX Ingress Controller.
+
+```
+# kubectl -n authorization get ingress
+NAME           CLASS   HOSTS                                 ADDRESS   PORTS     AGE
+proxy-server   nginx   csm-authorization.com,<other hosts>             80, 443   2m35s
+# kubectl -n authorization get service
+NAME                                               TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)                      AGE
+authorization-ingress-nginx-controller             LoadBalancer   10.105.85.204    <pending>     80:30162/TCP,443:32635/TCP   30s
```

Comment:
Can these values be zero'd out, similar to how its been done [here ](https://github.com/dell/csm-docs/blob/release-1.10.0/content/docs/deployment/helm/modules/authorization/_index.md#configuring-the-csm-authorization-proxy-server)